src/automata/automata.cpp

```diff
@@ -38,8 +38,8 @@ operator<<(std::ostream &os, const automata::ClockConstraint &constraint)
 }
 
 std::ostream &
-operator<<(std::ostream &                                                     os,
-           const std::multimap<std::string, const automata::ClockConstraint> &constraints)
```

Comment:
Why is the constness removed? I assume the general type has changed and probably this whole issue will be resolved by including strong types, right?

Reply:
Two reasons:
1. It does not really make sense to make all members const. If we want to have a const Transition, we can just declare the Transition const
2. For the proto parser, we need the assignment operator transitions. We cannot have assignment if the members are const.

Reply:
Good point, I agree!